source/prerequisites.adoc

```diff
@@ -17,13 +17,21 @@ explicitly to show which tools are used in this guide.
 endif::rhel[]
 
 ifdef::community[]
-On Fedora, CentOS 8, and RHEL 8:
+On Fedora, CentOS Stream 9, and RHEL 9:
 
 [source,bash]
 ----
 $ dnf install gcc rpm-build rpm-devel rpmlint make python bash coreutils diffutils patch rpmdevtools
 ----
 
+On CentOS Stream 8 and RHEL 8:
+
+[source,bash]
+----
+$ dnf install gcc rpm-build rpm-devel rpmlint make python3 bash coreutils diffutils patch rpmdevtools
```

Comment:
you are using python3 on rhel8 and python on rhel9. It should be the other way round, isn't it? The same in the other part bellow.

Reply:
`dnf install python` works on rhel9, because python-unversioned-command provides python.  I can adjust this to the actual package name if you like, it just makes the command longer for no real benefit.  Since the current instructions do not work on rhel8, I was trying to change as little as possible from the previous instructions but still get things working.  Longer term I'd like to send another pull request to switch the guide over to python3, which would simplify the instructions.

Reply:
Updated to use python3 on both rhel8 and rhel9.